File: server/src/game/tools/conf/output/ts/QuotesDataConfig.ts
class QuotesDataConfig {
    private static _dic:Dictionary<QuotesDataConfig> = new Dictionary<QuotesDataConfig>();
    private static _arr:Array<QuotesDataConfig> = new Array<QuotesDataConfig>();

	 public Id :number ;
     public TextId :number ;
    
     public KeyId :number ;
    

    
    public constructor(){

    }


    public static get Dic():Dictionary<QuotesDataConfig>{
        return  QuotesDataConfig._dic;
    }
    public static get Arr():Array<QuotesDataConfig>{
        return QuotesDataConfig._arr;
    }
    public static parse(json:any[]){

        for(var i = 0; i < json.length; i++){
            let config:QuotesDataConfig = <QuotesDataConfig>json[i];
            QuotesDataConfig._arr.push(config);
            QuotesDataConfig._dic.add(config.Id,config);
        }
    }
}

File: server/src/game/tools/conf/output/go/QuotesDataConfig.go
package conf

import (
	//"bufio"
	"encoding/json"
	"fmt"
	//"io"
	"io/ioutil"
	"os"
	//"strings"
)

type QuotesDataConfigElem struct {
	Id int
	TextId int
	Name string
	KeyId int
	
}

type QuotesDataConfig0 struct {
	Dic map[int]*QuotesDataConfigElem
	Arr []*QuotesDataConfigElem
}

var (
	QuotesDataConfig QuotesDataConfig0
)

func loadQuotesDataConfig(path string) {
	file, err := os.Open(path)
	defer func() {
		if file != nil {
			file.Close()
		}
	}()
	if err != nil {
		panic(fmt.Sprint("load config error, path=", path))
	}

	QuotesDataConfig.Dic = make(map[int]*QuotesDataConfigElem)
	QuotesDataConfig.Arr = make([]*QuotesDataConfigElem, 0, 32)

	bt,err  :=  ioutil.ReadAll(file)
	json.Unmarshal(bt, &QuotesDataConfig.Arr)
	for _,conf := range QuotesDataConfig.Arr{
		QuotesDataConfig.Dic[conf.Id] = conf
	}
	
}
